v-next/hardhat-ignition-viem/src/types.ts

```diff
@@ -19,6 +23,34 @@ export type IgnitionModuleResultsToViemContracts<
   >;
 };
 
+export interface ViemIgnitionHelper {
```

Comment:
I created a interface that can be exported as the public face of the IgnitionHelper. The impl moves into internal.
Note the interface is not generic in the chaintype.